frontend/src/tabs/zoom.pug

```diff
@@ -10,7 +10,14 @@
       v-bind:udate="info.untilDate",
       v-bind:avgsize="info.avgCommitSize")
 
-  .zoom__day(v-for="day in info.user.commits")
+  .zoom__week(v-if="filterTimeFrame === 'week'", v-for="week in info.user.commits")
```

Comment:
The `,` here should be unnecessary. Also, Vue strongly discourages using `v-if` and `v-for` together. If it is needed, please create another div for `v-if`